### PR TITLE
Adding a first blog post in a new Spotlight on DevOps series 

### PR DESCRIPTION
**Pull request by danavp**
Feedback?

### CODE REVIEW

File: _posts/2017-02-14-spotlight-on-devops-p1.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- culture
+- devops
+- Spotlight on DevOps
+---
+
+# Don’t just read about DevOps culture, play-test it!
+
+_This blog post is the first one in a new **Spotlight on DevOps** series._
+
+Have you ever tried to learn a new language by reading about it and not having an opportunity to practice? How fruitful was that experience?  Keep that in mind as you read on this invitation to experience DevOps culture through a simulation game!  Develop your DevOps transformation muscle memory and experience teams’ behavioral patterns.  Feel the difference DevOps culture makes in establishing trust and [psychological safety](https://en.wikipedia.org/wiki/Psychological_safety) in your organization.
```
Comment (jkenny) [inline _posts/2017-02-14-spotlight-on-devops-p1.md]: "... as you read this invitation ..."

File: _posts/2017-02-14-spotlight-on-devops-p1.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+My interest in DevOps originated from a very unlikely turn that my carrier took 7 years ago. An opportunity came up to push myself completely out of my comfort zone of a developer’s world. I’ve taken on a job of DBA Manager and found myself in a harsh, alerts-driven world of pagers, [disaster recoveries](https://en.wikipedia.org/wiki/Disaster_recovery) and escalation procedures. The sense of urgency and pressure was incredible and made me wonder, why I never knew about it as a developer…
```
Comment (jkenny) [inline _posts/2017-02-14-spotlight-on-devops-p1.md]: "career"

Comment (jkenny) [inline _posts/2017-02-14-spotlight-on-devops-p1.md]: "... comfort zone in a developer's world."

Comment (jkenny) [inline _posts/2017-02-14-spotlight-on-devops-p1.md]: "I'd taken on a job ..."

File: _posts/2017-02-14-spotlight-on-devops-p1.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+![Scrum Gathering Rio, Brazil - DevOps Culture simulation with Chocolate, Lego and Scrum Game](http://i.imgur.com/qKl4fWr.jpg?2) ![Build your T-shaped skills!](http://i.imgur.com/Q2tjPX7.jpg?1)
+![Agile Practitioners 2017, Israel - DevOps workshop with Chocolate, LEGO and Scrum game](http://i.imgur.com/QcYjpZ6.png?2)
+
+Every player in the game gets a special role to play individually as well as a part of his/her team.  Together players build products with LEGO, learn to respond to ever-changing market demand.  They wait for environments to be build by Operations, get interrupted by security upgrades and even get attacked by a hacker! The game engages everyone to the extent that they forget about time. They experience a range of emotions as they go through their DevOps journey and transition toward generative culture of collaboration and shared goals.
```
Comment (jkenny) [inline _posts/2017-02-14-spotlight-on-devops-p1.md]: "... with LEGO and learn ..."

Comment (jkenny) [inline _posts/2017-02-14-spotlight-on-devops-p1.md]: "... to be built by Operations, ..."

Comment (jkenny) [inline _posts/2017-02-14-spotlight-on-devops-p1.md]: "... transition towards a generative culture ..."

File: _posts/2017-02-14-spotlight-on-devops-p1.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+While this DevOps transformation is a gamified simulation, the lessons people learn are very real and can be applied to their actual DevOps transformations!
+Here are just a few examples of the “A-ha!” moments highlighted by the participants at [Scrum Gathering Porto](https://www.facebook.com/pg/Chocolate-Lego-and-Scrum-Game-Intro-to-DevOps-163199094149551/photos/?tab=album&album_id=169296690206458) and at [Lean-Agile practitioners of NJ](https://www.meetup.com/Lean-Agile-Practitioners-of-New-Jersey/events/235763877/) meetup:
+
+> “Even after DevOps transformation some Ops people want to keep being gate keepers. Hard to give up traditional roles!”
+
+> “Potentially shippable” not equals ”in production.”
```
Comment (jkenny) [inline _posts/2017-02-14-spotlight-on-devops-p1.md]: "... does not equal ..."

File: _posts/2017-02-14-spotlight-on-devops-p1.md [hunk 5]
```diff
@@ -14,9 +14,9 @@ tags:
 
 _This blog post is the first one in a new **Spotlight on DevOps** series._
 
-Have you ever tried to learn a new language by reading about it and not having an opportunity to practice? How fruitful was that experience?  Keep that in mind as you read on this invitation to experience DevOps culture through a simulation game!  Develop your DevOps transformation muscle memory and experience teams’ behavioral patterns.  Feel the difference DevOps culture makes in establishing trust and [psychological safety](https://en.wikipedia.org/wiki/Psychological_safety) in your organization.
+Have you ever tried to learn a new language by reading about it and not having an opportunity to practice? How fruitful was that experience?  Keep that in mind as you read this invitation to experience DevOps culture through a simulation game!  Develop your DevOps transformation muscle memory and experience teams’ behavioral patterns.  Feel the difference DevOps culture makes in establishing trust and [psychological safety](https://en.wikipedia.org/wiki/Psychological_safety) in your organization.
```
Comment (johncoghlan) [inline _posts/2017-02-14-spotlight-on-devops-p1.md]: I feel like we could eliminate this paragraph entirely and jump right into the post with the beginning of the narrative.

File: _posts/2017-02-14-spotlight-on-devops-p1.md [hunk 6]
```diff
@@ -28,14 +28,14 @@ The game starts with showcase of a large organization with departmental silos. D
...
+Every player in the game gets a special role to play individually as well as a part of his/her team.  Together players build products with LEGO and learn to respond to ever-changing market demand.  They wait for environments to be built by Operations, get interrupted by security upgrades and even get attacked by a hacker! The game engages everyone to the extent that they forget about time. They experience a range of emotions as they go through their DevOps journey and transition toward a generative culture of collaboration and shared goals.
 
 While this DevOps transformation is a gamified simulation, the lessons people learn are very real and can be applied to their actual DevOps transformations!
```
Comment (johncoghlan) [inline _posts/2017-02-14-spotlight-on-devops-p1.md]: If you remove the first paragraph, I'd expand on the lessons/takeaways that are taught by the game.

File: _posts/2017-02-14-spotlight-on-devops-p1.md [hunk 7]
```diff
@@ -14,7 +14,7 @@ tags:
 
 _This blog post is the first one in a new **Spotlight on DevOps** series._
 
-Have you ever tried to learn a new language by reading about it and not having an opportunity to practice? How fruitful was that experience?  Keep that in mind as you read this invitation to experience DevOps culture through a simulation game!  Develop your DevOps transformation muscle memory and experience teams’ behavioral patterns.  Feel the difference DevOps culture makes in establishing trust and [psychological safety](https://en.wikipedia.org/wiki/Psychological_safety) in your organization.
+A lot of people talk about DevOps Culture. Yes, you can learn about a culture by reading a book or a blog post. Much more effective and fun way to learn about a culture is by experiencing it. This blog post is _**your**_ invitation to experience DevOps culture through a simulation game!
```
Comment (johncoghlan) [inline _posts/2017-02-14-spotlight-on-devops-p1.md]: Third sentence: "A much more effective and fun way"

File: _posts/2017-02-14-spotlight-on-devops-p1.md [hunk 8]
```diff
@@ -14,7 +14,7 @@ tags:
...
+A lot of people talk about DevOps Culture. Yes, you can learn about a culture by reading a book or a blog post. Much more effective and fun way to learn about a culture is by experiencing it. This blog post is _**your**_ invitation to experience DevOps culture through a simulation game! 
 
 My interest in DevOps originated from a very unlikely turn that my career took 7 years ago. An opportunity came up to push myself completely out of my comfort zone in a developer’s world. I’d taken on a job of DBA Manager and found myself in a harsh, alerts-driven world of pagers, [disaster recoveries](https://en.wikipedia.org/wiki/Disaster_recovery) and escalation procedures. The sense of urgency and pressure was incredible and made me wonder, why I never knew about it as a developer…
```
Comment (johncoghlan) [inline _posts/2017-02-14-spotlight-on-devops-p1.md]: Drop comma and replace ellipsis with period in last sentence: "The sense of urgency and pressure was incredible and made me wonder why I never knew about it as a developer."

**Pull request merged**